rq/worker.py

```diff
@@ -160,11 +160,21 @@ def __init__(self, queues, name=None,
 
     def validate_queues(self):
         """Sanity check for the given queues."""
-        if not iterable(self.queues):
-            raise ValueError('Argument queues not iterable.')
         for queue in self.queues:
             if not isinstance(queue, self.queue_class):
-                raise NoQueueError('Give each worker at least one Queue.')
+                raise NoQueueError('{} is not a queue'.format(queue))
+
+    def process_queue_args(self, queue_args):
+        """ allow for a variety arguements to queues """
+        if isinstance(queue_args, text_type):
+            return self.queue_class(name = queue_args)
+        else:
+            queue_args = [queue_args] if isinstance(queue_args, text_type) else queue_args
```

Comment:
This line of code will never be used since we already checked for `text_text` instance in [Line 169](https://github.com/nvie/rq/pull/531/files#diff-1aac421e8c53e895ee40f3b7dc63144fR169)